sdk/storage/blobs/src/az_storage_blobs_blob_client.c

```diff
@@ -99,13 +86,12 @@ AZ_NODISCARD az_result az_storage_blobs_blob_upload(
   }
   (void)opt;
 
-  // check if url needs to be reset to initial state
-  AZ_RETURN_IF_FAILED(_az_reset_url_to_initial_state(client));
-
   // Request buffer
   // create request buffer TODO: define size for a blob upload
-  uint8_t url_buffer[1024 * 4];
+  uint8_t url_buffer[1024];
```

Comment:
Constant

Reply:
can't be constant.  We need to manipulate buffer content